core/emulate.c

```diff
@@ -232,16 +240,25 @@ static const struct em_opcode_t opcode_table[256] = {
 };
 
 static const struct em_opcode_t opcode_table_0F[256] = {
-    /* 0x00 - 0xAF */
+    /* 0x00 - 0x9F */
     X16(N), X16(N), X16(N), X16(N),
     X16(N), X16(N), X16(N), X16(N),
-    X16(N), X16(N), X16(N),
-    /* 0xB0 - 0xBF */
+    X16(N), X16(N),
+    /* 0xA0 - 0xAF */
+    X3(N),
+    F(em_bt, op_modrm_rm, op_modrm_reg, op_none, INSN_MODRM),
+    X7(N),
+    F(em_bts, op_modrm_rm, op_modrm_reg, op_none, INSN_MODRM),
     X4(N),
+    /* 0xB0 - 0xBF */
+    X3(N),
+    F(em_btr, op_modrm_rm, op_modrm_reg, op_none, INSN_MODRM),
     X2(N),
     I(em_movzx, op_modrm_reg, op_modrm_rm8, op_none, INSN_MODRM | INSN_MOV),
     I(em_movzx, op_modrm_reg, op_modrm_rm16, op_none, INSN_MODRM | INSN_MOV),
-    X4(N),
+    X2(N),
+    G(opcode_group8, op_modrm_rm, op_simm8, op_none, 0),
```

Comment:
I think the last parameter should be `INSN_MODRM` instead of 0. And that seems to be the only way to specify this flag: `em_decode_insn()` only checks this flag once, before it reads the flags of each member opcode. So setting it on each member of `opcode_group8` would have no effect.

Reply:
Oops, my error, `G` implies `INSN_MODRM`, sorry!